test_one.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+import requests
+from one import fetch
+from pytest_vcr import VCR
+
+
+def test_fetch():
+    url = "https://api.publicapis.org/entries"
+    try:
+        data = fetch(url)
+        assert data is not None
+    except Exception as e:
+        pytest.fail(f"error occurred : {str(e)}")
+
+
```

Comment:
This exception handling is not correct. Instead assert what type of error you are expecting from the method.
For ex: If you are passing wrong credentials, then it would be a authError. Hence you can use..
```
with pytest.raises(AuthError):
        assert fetch(url) is None 
```

Reply:
Implement with pytest.raises